float64[5] prior
---
float64[5] distribution
